src/System.Private.ServiceModel/src/System/ServiceModel/Channels/SslStreamSecurityUpgradeProvider.cs

```diff
@@ -303,139 +284,6 @@ protected override void OnEndOpen(IAsyncResult result)
         }
     }
 
-    internal class SslStreamSecurityUpgradeAcceptor : StreamSecurityUpgradeAcceptorBase
```

Comment:
UpgradeAcceptor's are used by Listener's, which we don't have in this code base, we only have client channel so only need upgrade initiators.